packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model/lib.esm.t

```diff
@@ -1,9 +1,8 @@
 ---
 to: "<%= rootDir %>/index.ts"
```

Comment:
we don't need this index file anymore. I think we should just not generate it at all and let users add their own index file if they want to.